modules/serverless-workflow-basic.adoc

```diff
@@ -0,0 +1,94 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="basic-workflow_{context}"]
+= Basic workflow using Knative Client
+
+Use this basic workflow to create, read, update, delete (CRUD) operations on a service. The following example deploys a link:https://github.com/knative/docs/tree/master/docs/serving/samples/hello-world/helloworld-go[simple Hello World service] that reads the environment variable `TARGET` and prints its output.
+
+
+.Procedure
+
+. Create a service in the `default` namespace from an image
++
+----
+$ kn service create hello --image gcr.io/knative-samples/helloworld-go --env TARGET=Knative
+
+Service 'hello' successfully created in namespace 'default'.
+Waiting for service 'hello' to become ready ... OK
+
+Service URL:
+http://hello.default.apps-crc.testing
+----
+
+. List service
```

Comment:
I know we went back and forth on the right thing to do with this, but if it's being done as a procedure, these steps should be full sentences with periods at the end. Even just "List the service." and "Curl the service endpoint.", etc.